app/chat_endpoint_handlers.py

```
@@ -125,16 +125,22 @@ async def chat_delete_success(chat_id, user_id):
 async def chat_history_success(chat_id, user_id):
     messages = []
     sql_connection.ping()
+    try:
+        sql_connection.cursor().execute(f"ALTER TABLE `{mariadb_name(user_id, chat_id)}`.messages ADD COLUMN IF NOT EXISTS image char(36) DEFAULT NULL;")
+    except mariadb.Error as e:
+        log_exception(e, "chat_history_success")
+        return {"messages": []}
     cursor = sql_connection.cursor()
     cursor.execute(
-        f"SELECT creator, content, aid FROM `{mariadb_name(user_id, chat_id)}`.messages ORDER BY aid;"
+        f"SELECT creator, content, image, aid FROM `{mariadb_name(user_id, chat_id)}`.messages ORDER BY aid;"
```

Ruff (0.13.1)

Possible SQL injection vector through string-based query construction

(S608)